models/models.js

```diff
@@ -5,3 +5,20 @@ exports.selectTopics = () => {
     return res.rows;
   });
 };
+
+exports.selectArticleById = (article_id) => {
+  return db
+    .query(
+      "SELECT author, title, article_id, body, topic, created_at, votes FROM articles WHERE article_id = $1;",
```

Comment:
any reason why you're not SELECTing * here?

Reply:
My mistake, the thinking process was that I might not need all keys so just started to list them in order.